### ISSUE TITLE
does not seem to work in chrome/ubuntu with evince pdf reader

### ISSUE THREAD

**Issue by JohnBoik** [andy-portmen/open-in-pdf-reader#3]
Hi. I cannot get the chrome extension to work with ubuntu and evince reader. I have put the path to evince (usr/bin/evince) in the Path to a PDF Reader executable box, and the path validates. However, regardless of the extension options I choose, there is no menu item for the extension on a right click of a pdf url. If I click on a pdf link, the pdf opens in chrome. Clicking the extension icon only results in the save pdf box popping up. If I click save, it simply saves the pdf. Evince does not open. Any suggestions on how to fix this?

**Comment by andy-portmen** [comment]
Try to open the DevTools of the background service worker of this extension (chrome://extensions/ -> developer mode) and see if you get anything useful in the console tab. If none, then close all Chrome instances and run it with `google-chrome --enable-logging` so that Chrome logs native messaging logs.

**Comment by JohnBoik** [comment]
Yes, I do get an error in the console. It occurs after I view a PDF in Chrome, then click on the "Open in PDF Reader" icon, then click on "Save". The error is:
```
worker.js:171 
{error: {…}, stdout: '', stderr: '/bin/sh: 1: start: not found\n'}
error
: 
{killed: false, code: 127, signal: null, cmd: 'start "/home/john/Downloads/2412.08688v1.pdf"'}
stderr
: 
"/bin/sh: 1: start: not found\n"
stdout
: 
```
And ideas on where to go from here?

**Comment by andy-portmen** [comment]
Also please share the options page of the extension. Do you have the absolute path of `evince pdf reader` on there? something like `/usr/bin/evince`? The extension needs to know how to find the reader.
